Documentation/CRDs/Object-Storage/ceph-object-store-crd.md

```diff
@@ -158,6 +158,10 @@ The [zone](../../Storage-Configuration/Object-Storage-RGW/ceph-object-multisite.
 The hosting settings allow you to host buckets in the object store on a custom DNS name, enabling virtual-hosted-style access to buckets similar to AWS S3 (https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html).
 
 * `dnsNames`: a list of DNS names to host buckets on. These names need to valid according RFC-1123. Otherwise it will fail. Each endpoint requires wildcard support like [ingress loadbalancer](https://kubernetes.io/docs/concepts/services-networking/ingress/#hostname-wildcards). Do not include the wildcard itself in the list of hostnames (e.g., use "mystore.example.com" instead of "*.mystore.example.com"). Add all the hostnames like openshift routes otherwise access will be denied, but if the hostname does not support wild card then virtual host style won't work those hostname. By default cephobjectstore service endpoint and custom endpoints from cephobjectzone is included. The feature is supported only for Ceph v18 and later versions.
+Please consider below points while setting this option:
+    * Domains added to `dnsNames` should listen to the same port mentioned in `CephObjectStoreSpec`.
+    * If the user specified both `port` and `securePort` in the `CephObjectStoreSpec`, then domains input into `dnsNames` must only specify `securePort`.
+    * If TLS is enabled with RGW, either TLS certificate include to verify the domains in `dnsNames` or user need to set `insecureSkipVerify` to true in the kubernetes secret.
```

Comment:
I can't tell if this section is intended to be part of the paragraph that begins with the bullet `* dnsNames` above, or if it is intended to be a separate paragraph.

As it is, the text "Please consider..." immediately follows the `dnsNames` bullet, which implies it should be the same content, but it also is indented on the left, which implies it should be a separate paragraph.

The 3 bullets are indented, which implies there is intent to have this be part of the `dnsNames` bullet.

IMO, this section is short enough that this will be easier to read for users if it is a separate paragraph. To format it as such, please add an empty line above "Please consider..." and un-indent the 3 bullets.